### ISSUE TITLE
remote images for timeline - suggestion/workaround

### ISSUE THREAD

**Issue by altsheets** [PanosSakkos/personal-jekyll-theme#46]
I am allowing external images in my timeline, then {{site.baseurl}} must not be prepended. My workaround is not the most elegant (that would be testing the variable "image" for "http(s)://") ... but it works. For remote images, I define a {{event.imageremote}} in [_config.yml](https://github.com/PanosSakkos/personal-jekyll-theme/blob/master/_config.yml#L70-L71)

    events:
      - image: "https://media.giphy.com....gif" 
        imageremote: True

and insert a test into [career.html](https://github.com/PanosSakkos/personal-jekyll-theme/blob/master/_includes/career.html#L22)

    <img class="img-circle img-responsive" src="{% unless event.imageremote %}{{ site.baseurl }}{%endunless%}{{ event.image }}" alt="">

so all the other (=local) images do get {{ site.baseurl }} prepended. 

It works, so I thought, why not share it with you guys. 

Many many thanks for all the work you put into this. My new site looks great!

**Comment by PanosSakkos** [comment]
I see, but I really believe that the images should be local, because if for example the remote image suddenly changes it can mess up your website, if for example is not a square :wink: 

Avoid taking dependencies, as always :wink: 

Thanks for the feedback! 
I had no idea that the repo would be used by so many people, and that's great!
Let's make the web a better place :wink:

**Comment by altsheets** [comment]
Thanks a lot! Yes, I get your point. Still, I want to link to external content. And already built that extension described above, anyways.

> I had no idea that the repo would be used by so many people, and that's great!  

Yes. Before I started, I compared a lot of different themes. And yours was closest to what I wanted to build. And beautiful, too. 

Here, have a look: My new website is launched - http://altsheets.ddns.net/assetgraphs/v2/products/ - using your theme as the basis, and then going wild on modifying it.

Thank you so much for your theme. I worked a lot on it, to adapt it to my needs, and fix some problems - but your underlying design was already so great, that it looks quite nice, I think. 

> Let's make the web a better place

Yes. And a happy place :-)
